package/MDAnalysis/core/universe.py

```diff
@@ -320,11 +339,18 @@ class Universe(object):
     .. versionchanged:: 2.5.0
         Added fudge_factor and lower_bound parameters for use with
         *guess_bonds*.
+
+    .. versionchanged:: 2.8.0
+        Added :meth:`~MDAnalysis.core.universe.Universe.guess_TopologyAttrs` API
+        guessing masses and atom types after topology
+        is read from a registered parser.
+
     """
     def __init__(self, topology=None, *coordinates, all_coordinates=False,
                  format=None, topology_format=None, transformations=None,
                  guess_bonds=False, vdwradii=None, fudge_factor=0.55,
-                 lower_bound=0.1, in_memory=False,
+                 lower_bound=0.1, in_memory=False, context='default',
+                 to_guess=('types', 'masses'), force_guess=(),
```

Comment:
```suggestion
                 to_guess=('types', 'masses'), force_guess=None,
```
Could you confirm that keeping `()` instead of `None` was intentional?
See: https://github.com/MDAnalysis/mdanalysis/pull/3753/#discussion_r1731835062

Reply:
@lilyminium feel free to turn this into an issue, we can fix this in post.

Reply:
yes it's intended to be that way indeed, for easier manuplation